recipes/Remi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "remi" %}
+{% set version = "2020.1.16" %}
+{% set styledName = "Remi" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "0a846809b3a6d10b6afd65cf6e74bfb855ffc165c74df80e29433174c4972a59"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+
+  requires:
+
+about:
+  home: "https://github.com/dddomodossola/remi"
+  license: "Apache 2.0"
+  license_family: "Apache"
+  license_file: "license.txt"
+  summary: "Remi is a GUI library for Python applications that gets rendered in web browsers. This allows you to access your interface locally and remotely."
```

Comment:
Please make the `summary` shorter, maybe just the first sentence. And you can use `description` for more details as you have now.